### DISCUSSION TITLE
Chatbot

### DISCUSSION THREAD

**Discussion by Nitaant** [alveusgg/alveusgg#386] [Ideas]
I was thinking maybe we could make chatbots of the ambassadors at alveus. It could be an interactive way of learning more about them.

**Comment by MattIPv4** [comment]
Could you perhaps provide some examples of conversations you might expect the chatbots to have?

My initial gut reaction to this is that while chatbots are sometimes a useful way to self-serve information, it feels to me like in this context we risk anthropomorphising the ambassadors and it also feels like we'd be burying a set of educational information behind chatbots that users would have to ask the right questions, rather than just having that educational information freely available immediately on the ambassador pages.

**Reply by briiree** [reply]
Does Alveus have a cow ambassador? 
How old is Winnie?
What is Winnie ambassador for?
Where did Winnie come from?
What does ccqad mean?

**Reply by MattIPv4** [reply]
I think all of these can be answered by the person spending any amount of time at all browsing the site, or simply asking chat. I don't see the need for an AI bot to do that.

**Reply by briiree** [reply]
Yeah - youtubers might not have options to ask chat, so that was my thought process

**Comment by 90sbb** [comment]
I like an anthropomorphized ambassador here and there. Imagine Sesame Street's BIg Bird... but it's Stompy.

**Comment by Nitaant** [comment]
I didn't even know what anthropomorphized means so I just removed myself
from the conversation. But yeah that's what I was thinking. Children love
asking questions.

**Comment by Nitaant** [comment]
We could always have the information available but like in the current context we always have stack overflow and documentations but I still prefer asking chatgpt.

**Reply by 90sbb** [reply]
Stompygpt
